src/command/restore/restore.c

```diff
@@ -1248,9 +1298,19 @@ restoreSelectiveExpression(Manifest *manifest)
             {
                 const String *db = strLstGet(dbList, dbIdx);
 
-                // Only user created databases can be zeroed, never system databases
-                if (cvtZToUInt64(strZ(db)) >= PG_USER_OBJECT_MIN_ID)
+                // Include system databases by default
+                if (strLstExists(excludeDbIdList, db) ||
+                    (strLstSize(includeList) > 0 && cvtZToUInt64(strZ(db)) >= PG_USER_OBJECT_MIN_ID))
```

Comment:
Not if we only use `--db-exlude`. I struggled a lot to figure this out but we won't remove the "to keep" databases from dbList if we don't use `--db-include`.
I'll simplify the IF regarding the system dbs but we'll have to keep it in case includeList is empty.

Reply:
Saying it all loud made me realize I could remove the IF by adding:
```
// Only exclude specified db in case no db to include has been provided
if (strLstSize(includeList) <= 0)
    dbList = strLstDup(excludeDbIdList);
```

Fixing the tests and I'll push the modification ;-)